include/lsst/jointcal/AstrometryFit.h

```diff
@@ -152,66 +110,7 @@ class AstrometryFit {
      */
     void offsetParams(const Eigen::VectorXd &delta);
 
-    /**
-     * Returns a chi2 for the current state.
-     *
-     * For the list of images in the provided  association and the reference
-     * stars, if any.
-     */
-    Chi2 computeChi2() const;
-
-    /**
-     * Contributions to derivatives from (presumably) outlier terms. No
-     * discarding done.
-     */
-    void outliersContributions(MeasuredStarList &mOutliers, FittedStarList &fOutLiers, TripletList &tList,
-                               Eigen::VectorXd &grad);
-
-    /**
-     * Discards measurements and reference contributions contributing to the chi2 more than nSigmaCut,
-     * computed as:
-     * @f[
-     *    <chi2> + nSigmaCut + rms(chi2)
-     * @f]
-     * (statistics over contributions to the chi2).
-     * No refit done.
-     *
-     * @param[in]  nSigmaCut  Number of sigma to cut on.
-     * @param[in]  measOrRef  Which outliers to remove. One of "Meas", "Ref" or "Meas Ref".
-     *
-     * @return     Total number of outliers that were removed.
-     */
-    unsigned removeOutliers(double nSigmaCut, const std::string &measOrRef = "Meas Ref");
-
-    /**
-     * Find Measurements and references contributing more than a cut, computed as
-     * @f[
-     *     <chi2> + nSigmaCut + rms(chi2).
-     * @f]
-     * The outliers are NOT removed, and no refit is done.
-     *
-     * After returning from here, there are still measurements that
-     * contribute above the cut, but their contribution should be evaluated after a
-     * refit before discarding them.
-     *
-     * @param[in]  nSigmaCut   Number of sigma to select on.
-     * @param[out] mSOutliers  list of MeasuredStar outliers to populate
-     * @param[out] fSOutliers  list of FittedStar outliers to populate
-     * @param[in]  measOrRef   Which outliers to remove. One of "Meas", "Ref" or "Meas Ref".
-     *
-     * @return     Total number of outliers that were removed.
-     */
-    unsigned findOutliers(double nSigmaCut, MeasuredStarList &mSOutliers, FittedStarList &fSOutliers,
-                          const std::string &measOrRef = "Meas Ref") const;
-
-    //! Just removes outliers from the fit. No Refit done.
-    void removeMeasOutliers(MeasuredStarList &outliers);
-
-    //! Just removes outliers from the fit. No Refit done.
-    void removeRefOutliers(FittedStarList &outliers);
-
-    //! Produces both ntuples (cook up names from the provided string)
-    void makeResTuple(const std::string &tupleName) const;
+    void saveResultTuples(const std::string &tupleName) const;
```

Comment:
I've gone and cleaned up a bunch of `const T &` things, and added a better docstring for that method. It's mostly for debugging, by dumping the all the current internal values to text files. We probably want something better eventually.